### PR TITLE
Updating to UFR Format

### PR DESCRIPTION
**Pull request by aaronbell**
I'm helping update repositories to Google's [United Font Repro](https://github.com/googlefonts/Unified-Font-Repository) format, as well as correcting STAT table issues in variable fonts. As an added benefit to this process, the font will now build automatically when PRs are accepted. 

Additionally, I made some minor updates to the font metadata / masters to get things to build properly. 

Once merged in, we'll submit an update to Google Fonts. Please let me know if you have any questions. Thanks!

### CODE REVIEW

**Comment by sursly** [comment]
hey @RosaWagner, does this sound good to you? Not trying to fact-check/gatekeep @aaronbell, I'm just as quick to defer to the experts :)

**Comment by RosaWagner** [comment]
@sursly, yes no worries 👍  we changed our specifications again and Aaron is aligning all the VF.
Thanks @aaronbell by the way :)

File: sources/config.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      - name: ExtraBold
+        value: 800
+      - name: Black
+        value: 900
+    - name: Italic
+      tag: ital
+      values:
+      - name: Italic
+        value: 1
+autohintTTF: False
```
Comment (RosaWagner) [inline sources/config.yaml:69]: @aaronbell we actually want the static ttf to be hinted. VF won't be auto-hinted even if this parameter is True (I think)

Comment (aaronbell) [reply sources/config.yaml:69]: Oh is that right? I had understood it to mean autohinting all ttf files (including variable). :/

Comment (RosaWagner) [reply sources/config.yaml:69]: I also have doubt so better check with the gftools guys, but from what I see in the other fonts I have build, only the statics gets autohinted

Comment (aaronbell) [reply sources/config.yaml:69]: Ugg. Well that means a lot of fixing for me!

Comment (RosaWagner) [reply sources/config.yaml:69]: Simon just confirmed we really can't hint VF with TTFautohint…

Comment (aaronbell) [reply sources/config.yaml]: Just updated. Should be resolved now.

**Comment by sursly** [comment]
Fantastic, Thank you both!

**Pull request merged**